ios/FluentUI/Pill Button Bar/PillButton.swift

```diff
@@ -133,4 +141,10 @@ open class PillButton: UIButton {
             }
         }
     }
+    
+    private func updateFont() {
+        if style == .transparent {
+            titleLabel?.font = isSelected ? Fonts.preferredFont(forTextStyle: .subheadline, weight: .bold) : Fonts.button1
```

Comment:
>lected ? Fonts.pr [](start = 35, length = 17)

can we add style methods, "selectedFont" and "font", like we have "selectedTitleColor" and "titleColor"?

Reply:
as this "font" and "selectedFont" are used only for .transparent case I didn't add separate style methods. Do you think we need them because they will increase the layer of complexity so instead of calling "Fonts.button1" we will call "font" that calls "Fonts.button1". Same for "selectedFont".

Reply:
I think the extra complexity is minor. It looks much more organized if we delegate all style specific code to the style enum.